### PR TITLE
feat: 🎸 new style for mobile pagination compact

### PR DESCRIPTION
**Pull request by mhogler**
After discussion with designers we decided to make the compact pagination buttons for mobile slightly smaller

### CODE REVIEW

File: src/Molecules/Pagination/MobilePaginationCompact.tsx [hunk 1]
```diff
@@ -3,26 +3,24 @@ import styled from 'styled-components';
...
   height: ${(p) => p.theme.spacing.unit(10)}px;
-  width: ${(p) => p.theme.spacing.unit(10)}px;
+  width: ${(p) => p.theme.spacing.unit(8)}px;
   border: none;
   padding: 0;
-  justify-content: center;
-  align-items: center;
   background-color: transparent;
   outline: none;
+  justify-content: ${(p) => (p.direction === 'left' ? 'flex-start' : 'flex-end')};
```
Comment (almightybruce) [inline src/Molecules/Pagination/MobilePaginationCompact.tsx:17]: this is only valid if flex-direction is row, and not row-reverse.. right?

Comment (almightybruce) [reply src/Molecules/Pagination/MobilePaginationCompact.tsx:17]: however maybe it's not possible to set flex-direction to anything other than row

Comment (mhogler) [reply src/Molecules/Pagination/MobilePaginationCompact.tsx:17]: You're right, it's only valid for row. I added` flex-direction: row;`, just to be explicit. Thanks!

**Review by toblu — APPROVED** [review]

File: src/Molecules/Pagination/MobilePaginationCompact.tsx (hunk 1 above)
Comment (gustav-pettersson-nordnet) [reply src/Molecules/Pagination/MobilePaginationCompact.tsx:18]: make sure direction prop is not leaking out in the DOM.

**Comment by gustav-pettersson-nordnet** [comment]
Will make a new one with the right title for new reviews.

**Pull request closed without merging**